vault/dendron.topic.schema.cook.first-schema.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 5U4eAiqshI67VxIL40KWH
+title: Making Your First Schema
+desc: 'Learn to harness the powers of schema and automatically insert templates into new notes'
+updated: 1640789533041
+created: 1640784965050
+---
+
+## Summary
+
+[[Schemas|dendron://dendron.dendron-site/dendron.topic.schema]] make management of new notes easier by automatically inserting Markdown [[templates|dendron://dendron.dendron-site/dendron.topic.templates]] into new notes. This guide will walkthrough making a template for daily journaling, using [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) method, and using schema to handle where the template will be used in your [[hierarchy|dendron://dendron.dendron-site/dendron.topic.hierarchies]].
+
+We'd like all daily journals to have a child note for our 5MJ:
+
+- `daily.journal.yyyy.mm.dd`: Logging tasks, and anything else we'd like in our daily journal. Dendron can automatically make these with the [[Daily Journal|dendron.topic.special-notes#daily-journal]] command (`Ctrl+Shift+I` / `Cmd+Shift+I`).
+- `daily.journal.yyyy.mm.dd.5mj`: Our 5MJ for the day. This is kept separate in order for `daily.journal.yyyy.mm.dd` templates to be maintained separately. If wanting to target these, information will be provided on how to.
+
+## Examples
+
+- [`templates` seed repository](https://github.com/dendronhq/templates)
+  - [5MJ Template](https://github.com/dendronhq/templates/tree/main/templates/templates.journal.5mj.md)
+
+## 5MJ Template
+
+### Optional: Add templates seed to your workspace
```

Comment:
cut this for now until we resolve the schema dependency issue

Reply:
also, the point of this tutorial is to make a schema from scratch so lets focus on that